### DISCUSSION TITLE
Video about WiCAN used to retrieve SOC (GERMAN only - sorry)

### DISCUSSION THREAD

**Discussion by camueller** [meatpiHQ/wican-fw#114] [General]
I created a video about using the WiCAN in order to make the SOC available to energy management system.
Software created is linked and available on Github.
The video is available in GERMAN only. I might consider creating an english version later.

https://youtu.be/MjC1eaUPTDg

**Comment by meatpiHQ** [comment]
@camueller This is awesome! Love it!

**Reply by meatpiHQ** [reply]
@camueller Would you like to try out the new [WiCAN Pro](https://www.crowdsupply.com/meatpi-electronics/wican-pro)?

If you want drop me an email here, I'll have some extra samples in a couple of weeks.
https://www.meatpi.com/contact
